### ISSUE TITLE
Remove the package manager section from the "Command-line install" page.

### ISSUE THREAD

**Issue by swedneck** [ipfs/ipfs-docs#589]
I feel that snap just causes more strangeness than is worth, and can really confuse people when they don't find stuff where they expect it (e.g. ~/.ipfs), and it can make helping the people using it more difficult.
I would suggest acknowleding that the snap package exists but not recommend it as THE way to install it on ubuntu.

It's also somewhat strange to only list chocolatey, homebrew, and snap, but that's a separate issue..

**Comment by johnnymatthews** [comment]
The purpose of including the package manager install options is to reduce the time and effort it takes to install IPFS. A lot of readers use these one-liners to quickly install IPFS, without much configuration.

Snap is included because it comes pre-installed with Ubuntu, and it's a pretty simple way to install IPFS. While Chocolatey and Homebrew don't come pre-installed on their respective operating systems, they are the de-facto package managers for those OSes.

**Comment by lidel** [comment]
@swedneck which docs you suggest changing/adjusting?
https://docs.ipfs.io/install/ipfs-desktop/#ubuntu suggests using `.deb` for Ubuntu (not `.snap`). 

On top of what @johnnymatthews  said, the downside is that `.deb` installed manually  won't get autoupdates and people will be stuck at old versions.  Snap users get updates for free, so perhaps we should suggest that instead, as long we solve `~/.ipfs` problem?

Two ideas/questions:
- Perhaps we should have additional version check running for platforms without autoupdate mechanism and display OS notification or something?
- What is the location of `~/.ipfs` in Snap, if not in user's home? Sounds like something we could fix, perhaps we are building `.snap` without some flag?

**Comment by ardunster** [comment]
I am not familiar with how IPFS is packaged, but as a Ubuntu user I'd try to install it through `apt` if at all possible rather than snap. Had too many weird issues with how snap handles things and don't want to deal with it. `apt` is at least as common, if not more so than snap, especially if people are running older LTSs.

**Comment by johnnymatthews** [comment]
> ... as a Ubuntu user I'd try to install it through `apt` ... - @ardunster 

As far as I know, the `apt` repository isn't as frequently maintained when compared to the `snap` package. Is this true @lidel, or am I making this up?

**Comment by lidel** [comment]
@johnnymatthews you are most likely right: apt is just a tool, it uses repo set up by distribution. Debian and Ubuntu use apt, but point at different repositories. 

Given that snap is published by our CI (https://github.com/ipfs/go-ipfs/pull/7529), and it is up to distribution maintainers/community to update the `.deb` package in apt repo to the latest version, snap will always be updated sooner than apt package. 

That being said, I don't like nor use snaps myself, and just like @ardunster prefer `apt` or Docker image.

**Comment by johnnymatthews** [comment]
Ok cool, thanks for clearing that up @lidel. It seems like the general sentiment is to get rid of Snap since it causes more problems than it fixes, and the alternative `apt` isn't too far out of date to be a major issue. I'll take Snap out :)

> For now I've filled #660 to add Docker, which was missing - @lidel 

Perfect! Thanks for that.

**Comment by johnnymatthews** [comment]
Hmm, we've got a tiny problem. IPFS isn't listed in any of the `apt` repos. With that in mind, I'm inclined to take the Package Manager section out of here. Chocolately and Homebrew aren't installed by default, Snap causes problems, and Apt doesn't contain IPFS packages. So all-in-all, none of these solutions are on-liners, which was the intention for this section. There are installation instructions for each OS under the Official distributions section, so we've got our bases covered there.
